src/main/java/net/dries007/tfc/common/blockentities/TFCBlockEntities.java

```diff
@@ -37,7 +37,8 @@
 {
     public static final DeferredRegister<BlockEntityType<?>> TILE_ENTITIES = DeferredRegister.create(ForgeRegistries.BLOCK_ENTITIES, MOD_ID);
 
-    public static final Collection<RegistryObject<Block>> SAPLING_LIST = TFCBlocks.WOODS.values().stream().map(map -> map.get(Wood.BlockType.SAPLING)).collect(Collectors.toList());
+    public static final Collection<RegistryObject<Block>> CHEST_LIST = TFCBlocks.WOODS.values().stream().map(map -> map.get(Wood.BlockType.CHEST)).collect(Collectors.toList());
+    public static final Collection<RegistryObject<Block>> TRAPPED_CHEST_LIST = TFCBlocks.WOODS.values().stream().map(map -> map.get(Wood.BlockType.TRAPPED_CHEST)).collect(Collectors.toList());
```

Comment:
There's no need to keep these around (or the `collect` / `stream` redundancy), they can be inlined into the block entity declarations.